src/app/view.component.ts

```diff
@@ -510,7 +511,54 @@ export class ViewComponent implements OnInit, AfterViewInit {
       obj = obj[key_location].children;
     }
   }
+  /* This function is used to load Precipitation Data in oneclick from TopDataset Button
+   */
+  LoadPrecipitation() {
```

Comment:
public/private?

Reply:
made it private as it is called within its component